examples/v3.0/json/petstore-expanded.json

```diff
@@ -0,0 +1,239 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Swagger Petstore",
+    "description": "A sample API that uses a petstore as an example to demonstrate features in the swagger-2.0 specification",
```

Comment:
`swagger-2.0 specification` should be changed to `OpenAPI 3.0 specification` as per PR #1373